smarts/core/plan.py

```diff
@@ -335,45 +335,41 @@ def create_route(self, mission: Mission):
 
         assert isinstance(self._mission.goal, PositionalGoal)
 
-        start_lane = self._road_map.nearest_lane(
+        start_lanes = self._road_map.nearest_lanes(
             self._mission.start.point,
-            include_junctions=False,
+            include_junctions=True,
         )
```

Comment:
Here, `radius` is not specified in call to `self._road_map.nearest_lanes()`. By default `radius=None` and thus the radius is set to `radius = self._default_lane_width` inside `self._road_map.nearest_lanes()`.

https://github.com/huawei-noah/SMARTS/blob/4482821e0fbf44d83b0f627738d7753666902a49/smarts/core/sumo_road_network.py#L792-L797